src/rtcModule/webrtc.cpp

```diff
@@ -2621,7 +2621,21 @@ void Call::onClientLeftCall(Id userid, uint32_t clientid)
         }
 
         SUB_LOG_DEBUG("ENDCALL received for ourselves, finishing the call");
-        destroy(TermCode::kErrNetSignalling, false);
+        auto wptr = weakHandle();
+        destroy(TermCode::kErrNetSignalling, false)
```

Comment:
There is a third param to indicate the reason of this Call being destroyed, which could be a great value while debugging. Please, add the reason of destruction here. Actually, that message is missing in many other calls, so you may want to improve it all over the code (about 20 calls).

Reply:
Ok, I'll add a description

Reply:
Fixed